redaxo/src/core/lib/util/path_default_provider.php

```diff
@@ -79,7 +79,7 @@ public function frontendController()
     public function backend($file)
     {
         if ($this->provideAbsolutes) {
-            return $this->base($this->backend . '/' . $file);
+            return $this->frontend($this->backend . '/' . $file);
```

Comment:
Ich denke ja. `backend()` meint den Pfad zum aufrufbaren Backend, nicht zu den Sourcen (`src()`), und das aufrufbare Backend muss innerhalb des Doc-Roots (= `frontend()`) liegen.

In meinem Experiment waren die Pfade dann so:
`base()` -> `/`
`src()` -> `/src`
`frontend()` -> `/public`
`backend()` -> `/public/redaxo`

In dem `backend()` Ordner liegt in dem Fall nur noch die Backend-index.php.